tests/integration_tests/test_visualization_api.py

```diff
@@ -92,14 +93,15 @@ def __init__(self, csv_filename):
         self.probability = predictions.iloc[:, 1:(num_probs + 2)].values
         self.ground_truth_metadata = self.model.training_set_metadata
         target_predictions = test_df[self.output_feature_name]
+        output_feature_hash = self.output_features[0][HASH]
         self.ground_truth = np.asarray([
-            self.ground_truth_metadata[self.output_feature_name]['str2idx'][
+            self.ground_truth_metadata[output_feature_hash]['str2idx'][
```

Comment:
this has been changed to be just NAME again ;)

Reply:
reverted back to use NAME